spec/parameters/_index.yml

```diff
@@ -34,6 +36,8 @@ heightQuery:
   $ref: './query/height.yml'
 idQuery:
   $ref: './query/id.yml'
+level0Query:
+  $ref: './query/level0.yml'
```

Comment:
would it be better to use words instead? e.g. `rootLevel` / `levelZero` etc..

Reply:
this is how it is in the database